ui/raidboss/data/04-sb/raid/o3s.js

```diff
@@ -417,6 +451,22 @@ export default {
         },
       },
     },
+    {
+      id: 'O3S The Queen\'s Waltz: Tethers You',
+      netRegex: NetRegexes.tether({ id: '0012' }),
+      infoText: (data, matches, output) => {
+        if (data.me === matches.target)
+          return output.breakTether({ player: data.ShortName(matches.source) });
+        else if (data.me === matches.source)
+          return output.breakTether({ player: data.ShortName(matches.target) });
```

Comment:
```suggestion
        const otherPlayer = data.me === matches.target ? matches.source : matches.target;
        return output.breakTether({ player: data.ShortName(otherPlayer) });
```

Reply:
Betrayed.  This was meant to only call out if it's on you.